File: CiscoPCCE.Toolkit/Bean/OrgSettingsBean.cs
using System.Xml.Serialization;

using CiscoPCCE.Toolkit.Sdk;

namespace CiscoPCCE.Toolkit.Bean
{
    [XmlType("OrgSettings")]
    public class OrgSettingsBean : BaseApiBean
    {
        [XmlElement("accessToken")]
        public string? AccessToken { get; set; }

        [XmlElement("idBrokerHost")]
        public string? IdBrokerHost { get; set; }

        [XmlElement("idHost")]
        public string? IdHost { get; set; }

        [XmlElement("orgID")]
        public string? OrgID { get; set; }
    }
}

File: JavaSrc/toolkit/bean/OrgSettingsBean.java
// ----------------------------------------------
package com.cisco.ccbu.cce.unifiedconfig.toolkit.bean;

import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiBean;
import javax.xml.bind.annotation.XmlSeeAlso;
import javax.ws.rs.Path;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.ReferenceBean;
import java.util.Date;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiListBean;

import javax.xml.bind.annotation.XmlElement;

import javax.xml.bind.annotation.XmlElementWrapper;


@XmlRootElement(name="OrgSettings")
public class OrgSettingsBean extends BaseApiBean {
  private String accessToken;
  private String idBrokerHost;
  private String idHost;
  private String orgID;

  public String getAccessToken() {
     return this.accessToken;
  }

  public void setAccessToken(String accessToken) {
     this.accessToken = accessToken;
  }

  public String getIdBrokerHost() {
     return this.idBrokerHost;
  }

  public void setIdBrokerHost(String idBrokerHost) {
     this.idBrokerHost = idBrokerHost;
  }

  public String getIdHost() {
     return this.idHost;
  }

  public void setIdHost(String idHost) {
     this.idHost = idHost;
  }

  public String getOrgID() {
     return this.orgID;
  }

  public void setOrgID(String orgID) {
     this.orgID = orgID;
  }


}
